cico_build_deploy.sh

```diff
@@ -59,9 +59,10 @@ setup_workspace
 
 cd $GOPATH/src/github.com/fabric8-services/fabric8-jenkins-idler
 echo "HEAD of repository `git rev-parse --short HEAD`"
+make login REGISTRY_USER=${DEVSHIFT_USERNAME} REGISTRY_PASSWORD=${DEVSHIFT_PASSWORD}
```

Comment:
Another key thing is that the login needs to be before the build, as the build may require images from the private repo and we need it to be authenticated.